patch/kernel/rockchip-rk3588-edge/1043-ROCK-5A-PCIe.patch

```diff
@@ -0,0 +1,10 @@
+diff --git a/arch/arm64/boot/dts/rockchip/rk3588s-rock-5a.dts b/arch/arm64/boot/dts/rockchip/rk3588s-rock-5a.dts
```

Comment:
There is already a patch 1023 adding pcie to rock5a, please modify that patch since this is not upstreamed.